### ISSUE TITLE
Audio can not be decoded since 1.36.1

### ISSUE THREAD

**Issue by Malacath-92** [microsoft/vscode#78333]
Short: The function `AudioContext.decodeAudioData()` fails in 1.36.1 but not in 1.35.1, given the same data.

- VSCode Version: 1.36.1-user and 1.37.0-insider
- OS Version: Windows 10

Steps to Reproduce:

1. Create extension with the following source:
```ts
import * as vscode from 'vscode';

let webviewPanel = vscode.window.createWebviewPanel(
    'extension_name',
    'Webview Name',
    vscode.ViewColumn.Two,
    {
        enableScripts: true
    }
);

export function activate(extensionContext: vscode.ExtensionContext) {
    let resourcePath = "vscode-resource:/path_to_some.mp3";

    webviewPanel.webview.html = `
<script type="text/javascript">
    const AudioContext = window.AudioContext || window.webkitAudioContext;
    const audioContext = new AudioContext();
    audioContext.resume();

    fetch('${resourcePath}')
        .then(response => {
            return response.arrayBuffer();
        })
        .then(buffer => {
            audioContext.decodeAudioData(buffer)
                .then(audioBuffer => {
                    let audio = audioContext.createBufferSource();
                    audio.buffer = audioBuffer;
                    audio.loop = true;

                    let analyser = audioContext.createAnalyser();
                    analyser.fftSize = 512;

                    audio.connect(analyser);
                    analyser.connect(audioContext.destination);
                    audio.start();
                })
                .catch(e => {
                    console.error("Error: " + e.message);
                });
        }).
        catch(e => {
            console.error("Error: " + e.message);
        });

    render();

    function render() {
        requestAnimationFrame(render);
    }
</script>
    `;
}
export function deactivate() {
    webviewPanel.dispose();
}
```
2. When the extension is activated, a webview panel will load, which will report `VM98:25 Error: Unable to decode audio data` in the Webview Developer Tools. Also the following warning is reported: `VM98:3 The Web Audio autoplay policy will be re-enabled in Chrome 70 (October 2018). Please check that your website is compatible with it. https://goo.gl/7K7WLu`
3. No audio plays.
4. Compare with behavior of same extension in 1.35.1, where no warning or  error is reported and audio plays.

**Comment by gjsjohnmurray** [comment]
Maybe a consequence of 1.36 being the first release using Chrome 69. On 1.35 you had Chrome 66.

The current Exploration build is piloting Electron 6 and gives you Chrome 76. See #76069

**Comment by Malacath-92** [comment]
@gjsjohnmurray I tried the exploration build and indeed it works again. For which version is Electron 6 targeted, so I can have a rough estimate as to when the feature will work again.

**Comment by gjsjohnmurray** [comment]
I don't know what the planned schedule is. Electron 6,0.0 [released two days ago](https://electronjs.org/blog/electron-6-0) and VSCode exploration builds using 6.0.0 [appeared yesterday](https://github.com/microsoft/vscode/pull/75802#issuecomment-516738642). Just beware that if you install that build and then check for updates you'll get downgraded to a build dated 2019-07-13.

Once #78131 is done (next week?) and 1.37 ships it'll be interesting to see if #75802 gets merged and appears in 1.38.0-insider.

**Comment by mjbvz** [comment]
Closing as upstream electron issue fixed by 6.0
